static/sass/_pattern_article.scss

```diff
@@ -0,0 +1,60 @@
+@mixin canonical-p-article {
+  .p-article {
+    margin-bottom: $spv--strip-shallow;
+
+    &__header,
+    &__row {
+        @extend %vf-row;
+    }
```

Comment:
What's the use of `.p-article__row` and `.p-article__header` if they just extend the row?

Reply:
My understanding is that when building a component, the bem notation isn't optional. So a row inside a component becomes `.p-component__row` - as for example in the navigation. @bartaz what's your recommendation on this?

Reply:
If `p-article__header` happens to use `row` styles, it's fine for it to extend the row under it's own name.

With `p-article__row` it seems that it could have been named differently, "row" suggests you can use columns there, but I don't think we do. If we create a semantic component like this, it shuold have semantic names. Maybe something like `p-article__content` (if that's what it contains?).